packages/angular_devkit/build_angular/src/dev-server/index.ts

```diff
@@ -120,11 +120,15 @@ export function serveWebpackBrowser(
     // In dev server we should not have budgets because of extra libs such as socks-js
     overrides.budgets = undefined;
 
+    // Disable output hashing for dev build as this can cause memory leaks
+    // See: https://github.com/webpack/webpack-dev-server/issues/377#issuecomment-241258405
+    overrides.outputHashing = OutputHashing.None;
```

Comment:
Can a warning be added if this is disabled so that users know that it is happening?

Reply:
Yeah I was actually going to do, but didn’t because bundle budgets is being overridden without warning.

Although I think for this case it would be better to issue one.